udata/tests/dataset/test_dataset_model.py

```diff
@@ -5,13 +5,14 @@
 from mongoengine import post_save
 
 from udata.models import db, Dataset
+from udata.core.dataset.factories import (ResourceFactory, DatasetFactory,
```

Comment:
I'm not sure it really saves a line and each new import will add a new line, you end up having some kind of flag code indentation which looks weird *to me*. 

When I made the project pep8 consistent, I tried to use the indentation I mentioned in my comment above so if there are still inconsistencies we have to fix those. My point is not to discuss one version vs. the other but just to be consistent and to chose the path of least resistance for that.